src/main/kotlin/no/nav/familie/ef/sak/beregning/barnetilsyn/BeregningBarnetilsynService.kt

```diff
@@ -91,6 +92,15 @@ class BeregningBarnetilsynService(private val featureToggleService: FeatureToggl
         brukerfeilHvis((kontantstøttePerioder.harPeriodeFør(innføringsMndKontantstøttefradrag))) {
             "Fradrag for innvilget kontantstøtte trår i kraft: $innføringsMndKontantstøttefradrag"
         }
+
+        brukerfeilHvis(utgiftsperioderDto.any { it.periodetype == null }) {
+            "Utgiftsperioder $utgiftsperioderDto mangler en eller flere aktivitetstyper"
+        }
+
+        brukerfeilHvis(utgiftsperioderDto.any{ it.aktivitetstype == null && it.erOpphørEllerSanksjon().not()}) {
+            "Utgiftsperioder $utgiftsperioderDto kan ikke ha en periode uten aktivitetstype dersom periodetypen ikke er opphør eller sanksjon"
+        }
```

Comment:
Er det tilstrekkelig å validere at den er ordinær? 
```suggestion
        brukerfeilHvis(utgiftsperioderDto.any{ it.aktivitetstype == null && it.periodetype == ORDINÆR}) {
            "Utgiftsperioden ${utgiftsperioderDto.fom /*eller noe slik*/} mangler aktivitetstype"
        }
```

Som nevnt litt lengre opp burde kanskje valideringen av periodetype og aktivitet ikke ære direkt koblet til beregning